src/model/src/centralized_io_module.F

```diff
@@ -1603,9 +1605,9 @@ subroutine retrieve_time_dep_gridded_data (jdate, jtime, vname)
 #ifndef mpas
```

Comment:
Again, this is if MPAS is NOT defined, so need to be careful. I do not think we should comment out all these lines in the ifdef and indef macro sections.  Please revisit.